fundamental-concepts/data-structures-stdlib/7_sets.md

```diff
@@ -0,0 +1,46 @@
+In the last step we added the `addresses` field to the `Person` data type.
+
+<pre>
+data Person = Person {
+  name: Text,
+  addresses: [Address],
+  age: Int
+}
+</pre>
+
+But does the order of the addresses matter to us? Shouldn't all addresses in each `Person` entry be
+unique? If we choose `addresses` to be a `Set` instead we have these two properties by
+construction:
+
+<pre>
+import DA.Next.Set
+
+data Person = Person {
+  name: Text,
+  addresses: Set Address,
+  age: Int
+}
+</pre>
+
+Every element of a set is unique and there is no order defined between the elements.
+
+`Set` is also a better choice for the container if you need to find, delete or check for membership
+often. `Set` is a worse choice if you find yourself iterating over big sets often. In that case
+you might want use a list.
+
+Take a look at the [documentation](https://docs.daml.com/daml/stdlib/DA-Next-Set.html) for the
+available algorithms operating on sets.
+
+>> How do check for membership of an element x in the set A ? <<
+[*] x `member` A`
```

Comment:
This infix notation needs to be explained somewhere.

Reply:
this has been added in the `functions` step.